README.md

```diff
@@ -11,18 +11,30 @@
 
 ### TL;DR
 
-We use this within our components to publish [StatsD](http://github.com/etsy/statsd) metrics from .NET code. We've been using this in most of our things, since around 2013.
+We use this library within our components to publish [StatsD](http://github.com/etsy/statsd) metrics from .NET code to a server. We've been using this in most of our things, since around 2013.
+
+### Supported platforms
+
+`JustEat.Statsd` version 4 is built for these target frameworks: `net451` , `netstandard2.0`, `netcoreapp2.1`, `netcoreapp2.2`.
 
 ### Features
 
-* StatsD metrics formatter
-* UDP client handling
+* Easy to use
+* Robust and proven
+* Tuned for high performance and low resource usage
+* Works well with modern .NET apps - .NET Core, .NET Standard 2,0 and `async ... await`
+* Supports standard statsd primitives: `Increment`, `Decrement`, `Timing` and `Gauge`. Supports sample rate.
+* Helpers to make it easy to time a `Func`, `Action` or a code block inside a `using` statement.
```

Comment:
Suggest changing to _"...a delegate, such as an `Action`, or a..."_ as `Func` isn't a real type.